src/MDCT.jl

```diff
@@ -63,4 +67,60 @@ end
 imdct(X::AbstractVector{T}) where {T<:Number} =
     imdct(copy!(fftwsimilar(X, size(X)), X))
 
+mutable struct MDCTPlan{T<:fftwNumber, N, inv} <: AbstractFFTs.Plan{T}
```

Comment:
Can just be called `Plan` since it is already in the `MDCT` namespace.

Reply:
Yeah if You prefer. I named it in the same way FFTW.FFTWPlan is named.